Source/Core/VideoCommon/TextureDecoder_Generic.cpp

```diff
@@ -170,20 +159,17 @@ static void DecodeDXTBlock(u32* dst, const DXTBlock* src, int pitch)
   if (c1 > c2)
   {
     // Approximation of x/3: 3/8 (1/2 - 1/8)
```

Comment:
This comment seems to be invalid now (as you now have 3/8 and 5/8 in the Util file).